internal/oci/uvm.go

```diff
@@ -332,11 +339,18 @@ func SpecToUVMCreateOpts(ctx context.Context, s *specs.Spec, id, owner string) (
 		case uvm.PreferredRootFSTypeVHD:
 			lopts.RootFSFile = uvm.VhdFile
 		}
+		if parseAnnotationsBool(ctx, s.Annotations, annotationDisableKernelDirectBoot, false) {
+			lopts.KernelDirect = false
+			lopts.KernelFile = uvm.KernelFile
```

Comment:
Dont get this line

Reply:
When the default opts are set, if there's [kernel direct support](https://github.com/microsoft/hcsshim/blob/99f064b70e88824d6653da19160d4ae02fd35a72/internal/uvm/create_lcow.go#L126), the default kernel file is set to the uncompressed kernel. Could also require that if the disable flag is set we also need to set the kernel file in the config, but I think this will be removed when the internal bug for direct boot is fixed.

Reply:
Oh I see. This is a const. I didnt realize. Can you just add a `// TODO:` That says when this should be removed.